xcodeproj/internal/output_files.bzl

```diff
@@ -112,6 +117,8 @@ def _create(
                 ))
         ],
     )
+    if dsym_files:
+        transitive_products = depset(transitive_products.to_list() + dsym_files.to_list())
```

Comment:
`to_list()` is very bas for performance. I think you just want to add the dsym files to the `depset(` creation above, as part of the array that is being sent into it.

Reply:
Should be resolved. There is still `to_list()`, but only for `dsym files`. Guess that's the only way.